addons/l10n_es_edi_facturae/tests/test_edi_xml.py

```diff
@@ -179,12 +194,12 @@ def test_in_invoice(self):
                 ],
             )
             invoice.action_post()
-            generated_file = invoice._l10n_es_edi_facturae_render_facturae()
-            self.assertTrue(generated_file)
-
-            with file_open("l10n_es_edi_facturae/tests/data/expected_in_invoice_document.xml", "rt") as f:
-                expected_xml = lxml.etree.fromstring(f.read().encode())
-            self.assertXmlTreeEqual(lxml.etree.fromstring(generated_file), expected_xml)
+            generated_file, error = invoice._l10n_es_edi_facturae_render_facturae()
+            if not error:
```

Comment:
Just `self.assertFalse(error)` instead of disabling the test if there is an error?